packages-exp/auth-exp/src/core/user/id_token_result.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { base64Decode } from '@firebase/util';
+
+import { IdTokenResult, ParsedToken } from '../../model/id_token';
+import { User } from '../../model/user';
+import { ProviderId } from '../providers';
+import { assert } from '../util/assert';
+
+export async function getIdTokenResult(
```

Comment:
this should be`_getIdTokenResult` per our style convention

Reply:
It's a public helper per the bit snippit line 142

Reply:
oh, my bad